### DISCUSSION TITLE
Data file not loading fully in react

### DISCUSSION THREAD

**Discussion by dhirajnitk** [emscripten-core/emscripten#17875] [Q&A]
I have used emscripten 2.0.10 to build a c++ application to js+wasm.  The application works perfectly with normal js application but If I use them in react app loading with script tag, the data file are not fully loaded to memory and idb.
The data file is 50 mb in size but once I load the js file in react i notice barely 2 kb of data in idb.

I don't wish to use import statement so  EXPORT_ES6=1 is not enabled in c++ build script.

I read a discussion on webpack issues with wasm so I am compiling the code with environment=web

Could someone suggest me what could be reason for data not fully loading into memory and idb cache

**Comment by DhirajEntropik** [comment]
Placing the data file in wrong location ( index html) fixed the issue so more of a bug of webpack where it
Couldn't find the data file
